src/Writers/HTMLWriter.jl

```diff
@@ -930,7 +929,8 @@ function get_url(ctx, path::AbstractString)
         end
         isempty(d) ? "index.html" : "$d/index.html"
     else
-        Formats.extension(:html, path)
+        # change extension to .html
+        string(splitext(path)[1], ".html")
```

Comment:
The other comment does not apply to this as the pattern is only used in this one place in `HTMLWriter`.

Reply:
What other comment?

Reply:
Oh, the review comment.